CIP-015x/format.cddl

```diff
@@ -38,14 +38,20 @@ chunk = {
 ; WIP is not considered as a fixed format in this version of CIP
 delta = {
   chunk_seq: uint .size 8,     ; sequence number
+  namespace: bstr,             ; namespace for the delta record
   chunk_format: uint,          ; 0 - raw, 1 - zstd, 2 - zstde
-  changes: bstr                ; [* entry] — list of entries, thumbstone allowed
+  changes: change_entry        ; [* entry] — list of entries, tombstone allowed
   footer: {
     entries_cnt: uint .size 4  ; count of entries
     chunk_hash: bstr .size 28  ; H(concat (digest e for e in entries))
   }
 }
 
+change_entry = tombstone_entry // delta_value_entry
+
+tombstone_entry = (0, ? key: bstr, ? value: bytes )
```

Comment:
Should we have a comment here about what a tombstone entry _without_ a key signifies? Nullifying the whole record?

Also, what does a non-null value indicate?